test/c3/appframework/appframework_test.go

```diff
@@ -91,7 +106,7 @@ var _ = Describe("c3appfw test", func() {
 			err := deployment.DeploySingleSiteClusterWithGivenAppFrameworkSpec(deployment.GetName(), indexerReplicas, true, appFrameworkSpec, 10)
```

Comment:
I dont see the `monitoringConsoleRef` added to the cluster. We should add it before deploying MC.